src/components/Integrations/BigQuery/ControlPanel/ControlPanel.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { InputSpanForm } from './InputSpanForm';
+
+export const ControlPanel = (props) => {
+  return (
+    <>
+      <div className="dp-container">
+        <div className="dp-rowflex p-t-48">
+          <div className="col-lg-5">
+            <InputSpanForm />
```

Comment:
The name **`InputSpanForm`** is weird. Maybe it can be changed to **`BigQueryAuthorizationForm`** or something similar

Reply:
Solved in next push

Reply:
Hecho, resuelvo la conversación...